### DISCUSSION TITLE
Search Kent C. Dodds blog

### DISCUSSION THREAD

**Discussion by Alarid** [johnlindquist/kit#600] [Share]
Because we all love [Kent's blog](https://kentcdodds.com/blog) so much, I made a small script to quickly search for a blog post 😁

[Install search-kent-blog](https://scriptkit.com/api/new?name=search-kent-blog&url=https://gist.githubusercontent.com/Alarid/210958c034bfeda6f1baf1cc29eed91e/raw/7777c4ed25cbbc5a4f7a5bd3bd9b7a50435e8ee1/search-kent-blog.js")

```js
/** @type {import("@johnlindquist/kit")} */
// Menu: Search Kent's Blog

const query = await arg("Query")
exec(`open https://kentcdodds.com/s/${encodeURIComponent(query)}`)

```

**Comment by johnlindquist** [comment]
```js
import "@johnlindquist/kit"
// Menu: Search Kent's Blog

const query = await arg("Query", async input => `
<div style="height:1280px">
  <iframe src="https://kentcdodds.com/s/${encodeURIComponent(input)}"  frameborder="0" style="height:100%;width:100%" height="100%" width="100%">
</div>
`)
```

Through the magic of iframes! lol @kentcdodds

![CleanShot 2021-11-18 at 09 04 00](https://user-images.githubusercontent.com/36073/142451921-49625818-163a-4cb8-b18e-93793dd8388d.gif)

**Reply by Alarid** [reply]
Oh my ! Even better 😁
